docs/configfile.adoc

```diff
@@ -0,0 +1,52 @@
+== Configuration file
+
+You can use smart-testing.y(a)ml file which is present at `${project.dir}/smart-testing.yml`.
```

Comment:
`smart-testing.y(a)ml` this doesn't look obvious in the doc. Specify both types or just write:

You can use smart-testing yaml file at `${project.dir}/smart-testing.yml` or with alternative suffix `${project.dir}/smart-testing.yaml`

Reply:
now I have noticed that it is mentioned here https://github.com/arquillian/smart-testing/pull/183/files#diff-1dfff5bec990830d634a922b323a0661R18 again